Makefile

```diff
@@ -0,0 +1,20 @@
+# Sane Makefile Preamble from https://tech.davis-hansson.com/p/make/
+SHELL := bash
+.ONESHELL:
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+
+ifeq ($(origin .RECIPEPREFIX), undefined)
+  $(error This Make does not support .RECIPEPREFIX. Please use GNU Make 4.0 or later)
+endif
+.RECIPEPREFIX = >
+#/preamble
+
+.PHONY: build
+build:
+>myurl=$$(curl -Ssl "https://api.github.com/repos/oscar-system/TutorialTesterforOscar/actions/workflows/CI.yml/runs?per_page=1" | jq  '.workflow_runs[0].jobs_url' | sed 's/"//g')
+#>echo $$myurl
+>curl -SsL $$myurl | jq '.["jobs"][1:] | .[] | .name+":"+.conclusion' | sed 's/^.*\s.*\s.*\s//' | sed 's/):/: /' | sed s'/"$$//' > _data/examples_status.yml
```

Comment:
It may well be that the GitHub fails to return a status (due to a temporary network issue, a temporary issue on their server, etc. ). Maybe tell `curl` to retry, it has a bunch of feature for that

```
     --retry <num> Retry request if transient problems occur
     --retry-all-errors Retry all errors (use with --retry)
     --retry-connrefused Retry on connection refused (use with --retry)
     --retry-delay <seconds> Wait time between retries
     --retry-max-time <seconds> Retry only within this period
```

Of course it still may fail even with retries. One could think about how to deal with that at some point, but it hopefully doesn't need to be a top priority right now.